akka-docs/src/main/paradox/scala/actors.md

```diff
@@ -905,6 +905,29 @@ Messages marked with `NotInfluenceReceiveTimeout` will not reset the timer. This
 `ReceiveTimeout` should be fired by external inactivity but not influenced by internal activity,
 e.g. scheduled tick messages.
 
+<a id="actors-timers"></a>
```

Comment:
Indeed there should be automatic anchors for all headings (though we don't currently check for dangling references, https://github.com/lightbend/paradox/issues/120